core/embed/rust/src/ui/model_tt/component/button.rs

```diff
@@ -247,11 +247,6 @@ where
             }
             Event::Touch(TouchEvent::TouchMove(pos)) => {
                 match self.state {
-                    State::Released if self.area.contains(pos) => {
-                        // Touch entered our area, transform to `Pressed` state.
-                        self.set(ctx, State::Pressed);
-                        return Some(ButtonMsg::Pressed);
-                    }
```

Comment:
Without this transition, does it still make sense to distinguish Released from Initial?

Reply:
One thing i see is that Released -> Initial transition clears long timer when touch ends outside of buttons area. While it would work to clear the timer even in case when the button wasn't pressed, not sure if its cleaner solution